client/src/components/SpotifyContent/SpotifyContent.tsx

```diff
@@ -56,32 +59,26 @@ function SpotifyContent({ headerBackground }: HeaderBkgdType) {
         dispatch(setPlaylistSongs(songData)); // set song data from selected playlist
       }
     };
-    if (playlist.selectedPlaylist) {
+    if (selectedPlaylist) {
       getPlaylistSongs();
     }
-  }, [dispatch, playlist.selectedPlaylist]);
-
-  useEffect(() => {
-    if (playlist.playlistSongs) {
-      console.log(playlist.playlistSongs.tracks);
-    }
-  });
+  }, [dispatch, selectedPlaylist]);
 
   return (
     <ContentContainer>
-      {playlist.playlistSongs && (
+      {Object.keys(playlistSongs).length > 0 && (
```

Comment:
Can this be refactored to avoid `Object.keys()`?

```suggestion
      {playlistSongs.length && (
```

Reply:
after destructuring playlistSongs, i ended up using 'name' as the initial condition